test/e2e/service_cleanup.py

```diff
@@ -86,19 +86,22 @@ def service_cleanup(config: dict):
     )
 
     try:
-        delete_sns_topic(resources.SnsTopicARN)
+        delete_sns_topic(resources.SnsTopic1)
+        delete_sns_topic(resources.SnsTopic2)
     except:
-        logging.exception(f"Unable to delete SNS topic {resources.SnsTopicARN}")
+        logging.exception(f"Unable to delete SNS topics {resources.SnsTopic1}, {resources.SnsTopic2}")
 
     try:
-        delete_security_group(resources.SecurityGroupID)
+        delete_security_group(resources.SecurityGroup1)
+        delete_security_group(resources.SecurityGroup2)
     except:
-        logging.exception(f"Unable to delete VPC Security Group {resources.SecurityGroupID}")
+        logging.exception(f"Unable to delete VPC Security Groups {resources.SecurityGroup1}, {resources.SecurityGroup2}")
```

Comment:
Please have `try: except:` block for each individual `delete_security_group` call.